src/action/auth-mobile.js

```diff
@@ -75,6 +75,8 @@ class AuthAction {
     const { auth } = this._store;
     if (auth[param]) {
       auth[param] = auth[param].slice(0, -1);
+    } else if (param === 'newPin' && this._store.walletUnlocked) {
+      this._nav.goSettings();
```

Comment:
Same comment here. If we had custom mobile views, we could have a back button in the header to cancel and go back to setting (like we do on desktop).